Doc/library/uuid.rst

```diff
@@ -216,6 +220,16 @@ The :mod:`uuid` module defines the following functions:
 
 .. index:: single: uuid5
 
+
+.. function:: uuid8(a=None, b=None, c=None)
+
+   TODO
```

Comment:
Please write some documentation :-)

Reply:
I deliberately left it as a todo until someone had a look at the implementation and in case we were to change the standard implementation (the RFC gives quite a lot of freedom on how to implememt it). 

I will write down the documentation when I'm back!